src/Relation/Binary/Domain/Structures.agda

```diff
@@ -0,0 +1,79 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Structures for domain theory
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Domain.Structures where
+
+open import Data.Product using (_×_; _,_; proj₁; proj₂)
+open import Function using (_∘_)
+open import Level using (Level; _⊔_; suc)
+open import Relation.Binary.Bundles using (Poset)
+open import Relation.Binary.Domain.Definitions
+
+private variable
+  a b c ℓ ℓ₁ ℓ₂ : Level
+  A B : Set a
+
+
+module _ {c ℓ₁ ℓ₂ : Level} (P : Poset c ℓ₁ ℓ₂) where
```

Comment:
Do these definitions really need a `Poset` or just a relation? I don't immediately see the use of any of the other fields of `Poset` below.

Reply:
Looping back to @TOTBWF 's question re poset completion: no. Please consider adding to `stdlib`!
As for 'not buying much' between `Preorder` and `Poset`, we *tend* to try to state (and prove!) things wrt maximum-sensible generality, and antisymmetry feels like 'noise' much of the time in developing the basics... but YMMV.